packages/ds-medicare-gov/docs/src/pages/patterns/consistent-header.md

```diff
@@ -0,0 +1,125 @@
+  - Default
+    - The header will detect whether or not the current page url matches any of the links in the Consistent Header. If there is a match, then the underline will remain on that CH menu selection after navigating to that new page.
+      - Bene clicks on Basics > Your Medicare Costs and navigates to https://imp.medicare.gov/your-medicare-costs
+      - On https://imp.medicare.gov/your-medicare-costs the Basics menu will be underlined
+    - If a page doesn’t match any of the links in the Consistent Header, then no underline will be shown in the menu.
+    - Passing active-nav-section configuration will override the default
+  - Manual configuration - Application teams can manually set where the underline show display in the Consistent Header to aid in navigation
+    - Pass the active-nav-section configuration AND the specific menu for the underline to show
+      - Consistent Header menu options are: “"basics", "health & drug plans", "providers & services", or "account"
+    - Passing active-nav-section configuration will override the default
+
+### Code Examples
+
+#### HTML
+
+The following example shows what the div element should look like before importing the scripts:
+
+`<div id="root" lang="en" lang-toggle-link="your-language-toggle-link-here" login-url="login-url" logout-url="logout-url" enable-live-chat="true">`
+
+#### JS
+
+Implementers should load the JavaScript and CSS files programmatically from the asset-manifest.json in order to get the latest versions whenever they become available. Here's an example of how to do that in the imp environment, using jQuery.
+
+````
+loadFilesFromManifest('https://frontend.imp.medicare.gov/', 'asset-manifest.json');
+
+//Gets and parses the entrypoints node of the asset manifest file
+function loadFilesFromManifest(rootURL, manifest) {
+	var manifestURL = rootURL + manifest;
+	$.getJSON(manifestURL, function(json) {
+		importFilesFromArray(json.entrypoints, rootURL);
+	});
+}
+
+//Takes the provided array, parses it and loads
+// all of the JS and CSS files based on their file extension
+function importFilesFromArray(files, rootURL) {
+	for (var i = 0; i < files.length; i++) {
+		if (files[i].endsWith('.js'))
+			importJSFile(rootURL + files[i]);
+		else if (files[i].endsWith('.css'))
+			importCSSFile(rootURL + files[i]);
+	}
+}
+
+//Imports a single JS file
+function importJSFile(url) {
+	var jsElement = document.createElement('script');
+	jsElement.src = url;
+	jsElement.type = 'text/javascript';
+
+	var head = document.head;
+	head.insertBefore(jsElement, head.childNodes[0]);
+}
+
+//Imports a single CSS file
+function importCSSFile(url) {
+	var cssElement = document.createElement('link');
+	cssElement.href = url;
+	cssElement.rel = 'stylesheet';
+	cssElement.type = 'text/css';
+
+	var head = document.head;
+	head.insertBefore(cssElement, head.childNodes[0]);
+}```
```

Comment:
This should be four ticks in order to format properly on the doc site

Current rendering
<img width="1657" alt="Screen Shot 2022-03-07 at 11 55 06 AM" src="https://user-images.githubusercontent.com/33579665/157099364-44383bc3-524c-453f-be3a-b78ce487e61f.png">